app/facades/foods_facade.rb

```diff
@@ -0,0 +1,13 @@
+require './app/services/fdc_services'
+
+class FoodsFacade
+  def food_results(query)
+    response = FdcService.new.food_search(query)
+    foods_hash = response[:foods].first(10)
+    foods_hash.map { |hash| Food.new(hash) }
+  end
```

Comment:
You can pick out the total number of foods as well as get the food objects returned at the same time. You're building out a new method right now to handle getting the total number of results, but it comes back with the response you have on line 5. You could return both variables in an array 
`[total_count,foods]`